Microsoft.Toolkit.Uwp.SampleApp/Shell.xaml.cs

```diff
@@ -297,8 +298,8 @@ public void RegisterNewCommand(string name, RoutedEventHandler action)
                     HideInfoArea();
                 }
 
-                TitleTextBlock.Text = $"{category.Name} -> {_currentSample.Name}";
-                ApplicationView.SetTitle(this, $"{category.Name} - {_currentSample.Name}");
+                TitleTextBlock.Text = $"{category.Name} -> {_currentSample?.Name}";
```

Comment:
_currentSample can't be null here it's checked on line 230, no?

Reply:
I thought so, but it still threw an exception and crashed anyway. There are a few awaits in between the beginning and end of this function. My guess is that I triggered a change in the sample before it was finished and then caused the crash. I thought to add a lock, but you can't await in a lock. So this seemed fair enough

Reply:
hehe, you can lock around an await with [AsyncEx](https://github.com/StephenCleary/AsyncEx)!  I use it in the Monaco Editor component, it's great.  So the Sample App already has a dependency on it.  If you think you need this, you could add something around it, but if the ? fixes it for now, it's probably easiest.